R/setup.R

```diff
@@ -0,0 +1,55 @@
+
+#' Create a JSON config file
+#'
+#' @param apikey An \emph{Access Token} provided by Pushbullet (see details). If not
+#'   provided in the function call, the user will be prompted to enter one.
+#' @param config_file A string giving the path where the configuration file will
+#'   be written. RPushbullet will automatically attempt load
+#'   \code{~/.rpushbullet.json} (the default)
+#'
+#' @details This function writes a simple default configuration file based
+#'   on a given apikey.  It is intended to be run once to help new users setup
+#'   RPushbullet. Running multiple times without overriding the
+#'   \code{config_file} parameter will overwrite the default file. An \emph{Access
+#'   Token} may be obtained for free by logging into the Pushbullet website,
+#'   going to \url{https://www.pushbullet.com/#settings}, and clicking on
+#'   "Create Access Token".
+#'
+#' @export
+#' @importFrom RJSONIO toJSON
+#'
+#' @examples
+#' \dontrun{
+#'
+#' # Interactive mode.  Just follow the prompts.
+#' pbSetup()
+#' }
+#' @author Seth Wenchel
+pbSetup <- function(apikey=NULL, config_file="~/.rpushbullet.json"){
+  if(is.null(apikey)){
+    key <- readline("Please enter your API key (aka 'Access Token': ")
```

Comment:
The formatting is an easy fix.

json issue is OBE (#32)

The last one is up to you.  If you merge this and put it on CRAN, you are signing up to maintain it in some sense (though I'd imagine you'd just tag me to fix any issues).  You could put it in the (currently empty) wiki, but users would have to know to look there; vignettes or README is probably more helpful.  But even then you still have to maintain it.

My objective with this function is similar to implementing the changes for CRAN curl package, namely lowering the bar of entry to users ("reducing the friction" as the cool kids say).  So it's generally only once per user, but it might save you from getting a support request email.

Just let me know.  I can fix this up or you can copy it into documentation.